components/ProgressView/src/MDCProgressView.m

```diff
@@ -292,7 +292,9 @@ + (UIColor *)defaultTrackTintColorForProgressTintColor:(UIColor *)progressTintCo
 
 - (void)updateProgressView {
   // Update progressView with the current progress value.
-  CGFloat progressWidth = MDCCeil(self.progress * CGRectGetWidth(self.bounds));
+  CGFloat scale = self.window.screen ? self.window.screen.scale : 1;
```

Comment:
Tip: Treating a CGFloat as a Boolean seems pretty risky. 

Suggestion: 

```objc
CGFloat scale = self.window.screen.scale > 0 ? self.window.screen.scale : 1;
```

Reply:
Done, though to clarify: I was not using CGFloat as a Boolean here; I was testing if the screen is nil before making use of it. The > 0 check on the scale is safer in case the screen scale is somehow 0, so I've switched to that.

Reply:
Ah, yes. I clearly need to increase my coffee dosage...  ☕️ 📈